src/library_gl.js

```diff
@@ -614,6 +614,17 @@ var LibraryGL = {
         }
       }
 
+      context.disjointTimerQueryExt = GLctx.getExtension("EXT_disjoint_timer_query");
+      if (context.disjointTimerQueryExt) {
+        GLctx['createQueryEXT'] = function() { return context.disjointTimerQueryExt['createQueryEXT'](); };
```

Comment:
Let's keep only the single line `context.disjointTimerQueryExt = GLctx.getExtension("EXT_disjoint_timer_query");` from this block. That is, obtain the extension but not do the routing functions, since they are a bit redundant.